### ISSUE TITLE
RuleTile with Random GameObject instead of Sprite

### ISSUE THREAD

**Issue by superfrantv** [Unity-Technologies/2d-extras#320]
Hello, 

i created differend Grass-Prefabs wiht Animation and so on, i will place them using a RuleTile's GameObject Field, but i need the GameObject Field there where actually the Random Sprites Field is.

**Comment by ChuanXin-Unity** [comment]
I'm not sure what you mean by that. Do you want to have random GameObjects instead of random Sprites for the RuleTile?

**Comment by superfrantv** [comment]
Yes thats my idea a Tile where GameObjects instead of random Sprites can selected.

**Comment by ChuanXin-Unity** [comment]
I added some changes to the RuleTile at
https://github.com/Unity-Technologies/2d-extras/commit/ae7a3285cc3b1020789d6dbda77fd0e79368d344 which you can refer to for doing this.
